awx/api/serializers.py

```diff
@@ -1333,6 +1333,8 @@ def validate(self, attrs):
             scm_type = attrs.get('scm_type', u'') or u''
         if self.instance and not scm_type:
             valid_local_paths.append(self.instance.local_path)
+        if scm_type and self.instance.local_path != attrs.get('local_path'):
```

Comment:
Regarding failing tests, you might want something like this:

```python
if scm_type and "local_path" in attires and and self.instance.local_path != attrs['local_path']:
```

Reply:
:+1: tested locally.